Sources/TuistDependencies/SwiftPackageManager/SwiftPackageManagerInteractor.swift

```diff
@@ -161,16 +161,14 @@ public final class SwiftPackageManagerInteractor: SwiftPackageManagerInteracting
             isLegacy: isLegacy,
             packageManifestFolder: packageManifestPath.removingLastComponent()
         )
+        let content: String;
         switch manifest {
-        case let .content(content):
-            try fileHandler.write(content, path: packageManifestPath, atomically: true)
+        case let .content(inputContent):
+            content = inputContent
         case .manifest:
-            if fileHandler.exists(packageManifestPath) {
-                try fileHandler.replace(packageManifestPath, with: pathsProvider.sourcePackageSwiftPath)
-            } else {
-                try fileHandler.copy(from: pathsProvider.sourcePackageSwiftPath, to: packageManifestPath)
-            }
+            content = try fileHandler.readTextFile(packageManifestPath)
         }
+        try fileHandler.write(content, path: packageManifestPath, atomically: true)
```

Comment:
Can we skip the process of writing another `Package.swift`?

Reply:
I don't think so, if we want SPM resolution and folders to be created in `Tuist/Dependencies/SwiftPackageManager`, then the Package should be located there.
It's more conveniente for those to be in a separate folder, so you can just gitignore the whole `Tuist/Dependencies/SwiftPackageManager`

Reply:
Does it need to be `Tuist/Dependencies/SwiftPackageManager`? We adopted that structure because we supported multiple package managers so I'm wondering if we should simplify things a bit and stick to SPM's convention.

Reply:
Having everything in `Tuist/` would be strange I think. Anyway, it would be a breaking change and in case it would go in 4.0